scripts/EN/Harry

```diff
@@ -6144,87 +6144,87 @@ CleanScreen(); // 0x7c12: 00000064
-ShowText((1734)"¥"Your role is over. We don't need you anymore. What do you think you can accomplish by coming here?¥""); // 0x7d4e: 0000100c
+ShowText((1734)"¥"Your role is over.¥nW2We don't need you anymore.¥nW2What do you think you can accomplish by coming here?¥""); // 0x7d4e: 0000100c
 HandleInput(); // 0x7d54: 00000010
-ShowText((1735)"A smile appears on Kaufmann's face and he reaches into his pocket. ¥"My, aren't we getting cocky? Bet you can't see this and keep your cool.¥""); // 0x7d58: 0000100c
+ShowText((1735)"A smile appears on Kaufmann's face and he reaches into his pocket.¥n¥nW2¥"My, W1aren't we getting cocky?¥nW1Bet you can't see this and keep your cool.¥""); // 0x7d58: 0000100c
 HandleInput(); // 0x7d5e: 00000010
 CleanScreen(); // 0x7d62: 00000064
 LoadBackground(0xcc); // 0x7d66: 00001069
 DrawBackground(); // 0x7d6c: 00000065
-ShowText((1736)"The item that Kaufmann pulled out of his pocket... It is definitely the small bottle that I found in the gas tank of the motorcycle that was in that garage somewhere."); // 0x7d70: 0000100c
+ShowText((1736)"The item that Kaufmann pulled out of his pocket…¥nW2It is definitely the small vial that I found in the gas tank of the motorcycle W1that was in that garage somewhere."); // 0x7d70: 0000100c
 HandleInput(); // 0x7d76: 00000010
-ShowText((1737)"¥"Aglaophotis! I thought I got rid of that!¥" ¥"All I had to do was plant it somewhere for you to find. You, all, well, it kept you busy. Ha! You're easy! And there's more where this came from!¥""); // 0x7d7a: 0000100c
+ShowText((1737)"¥"Aglaophotis! W2I thought I got rid of that!¥"¥n¥nW2¥"All I had to do was plant it somewhere for you to find.¥nW1You, all, well, it kept you busy.¥nW2Ha! You're easy! And there's more where this came from!¥""); // 0x7d7a: 0000100c
 HandleInput(); // 0x7d80: 00000010
 CleanScreen(); // 0x7d84: 00000064
 LoadBackground(0x0); // 0x7d88: 00001069
 DrawBackground(); // 0x7d8e: 00000065
-ShowText((1738)"¥"Stop it!!¥" Kaufmann throws the small bottle at Alessa. All four of us watch the bottle fly in a beautiful parabola, our eyes fixed on the sacred liquid within."); // 0x7d92: 0000100c
+ShowText((1738)"¥"Stop it!!¥"¥n¥nW2Kaufmann throws the small vial at Alessa.¥nW2All four of us watch the vial fly in a beautiful parabola, our eyes fixed on the sacred liquid within."); // 0x7d92: 0000100c
 HandleInput(); // 0x7d98: 00000010
-ShowText((1739)"There is tension. Although she is normally calm and composed, that drug is enough to frighten Dahlia. That small bottle hit a part of Alessa's body, shattered into pieces, and soaked Alessa uncomfortably with the red liquid that was inside."); // 0x7d9c: 0000100c
+ShowText((1739)"There is tension.¥nW2Although she is normally calm and composed, that drug is enough to frighten Dahlia.¥nW2The small vial superbly strikes part of Alessa's body W1and shatters, W1uncomfortably showering her with the red liquid held within and staining part of her white clothing red."); // 0x7d9c: 0000100c
```

Comment:
Changed bottle to vial to match PS1 and then modified the entire last sentence to better capture the meaning from the Japanese.